pkg/chart/airflow.go

```diff
@@ -436,3 +427,30 @@ func (c Client) getOrGeneratePassword(ctx context.Context, teamID, key string, g
 
 	return "", fmt.Errorf("a %v exisits for %v, but it's empty or doesn't belong to Airflow", key, teamID)
 }
+
+func (c Client) createAirflowHelmEvent(ctx context.Context, teamID string, eventType database.EventType, logger logger.Logger) error {
```

Comment:
```suggestion
func (c Client) registerAirflowHelmEvent(ctx context.Context, teamID string, eventType database.EventType, logger logger.Logger) error {
```
Bare så det matcher navngivingen vi har på det andre med Events.